DataFormats/EgammaReco/interface/ElectronSeed.h

```diff
@@ -1,14 +1,38 @@
-#ifndef ElectronSeed_h
-#define ElectronSeed_h 1
-
-/** \class reco::ElectronSeed
- *
- * ElectronSeed is a seed for gsf tracking,  constructed from
- * either a supercluster or a ctf track.
- *
- * \author D.Chamont, U.Berthon, C.Charlot, LLR Palaiseau
- *
- ************************************************************/
+#ifndef DataFormats_EgammaReco_ElectronSeed_h
```

Comment:
yes it is. 

Unrelated to this there is a lot of other phaseI stuff needed to be compiled (its not just this E/gamma change which creates the situation David was complaining about).